docs/cmdline-opts/vlan-priority.md

```diff
@@ -0,0 +1,21 @@
+---
+c: Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
+SPDX-License-Identifier: curl
+Long: vlan-priority
+Arg: <priority>
+Help: Set VLAN priority
+Added: 8.9.0
+Category: connection
+Protocols: All
+Multi: single
+See-also:
+  - ip-tos
+Example:
+  - --vlan-priority 4 $URL
+---
+
+# `--vlan-priority`
+
+Set VLAN priority as defined in IEEE 802.1Q. (Added in 8.9.0).
```

Comment:
Maybe mention that this sets a field on Ethernet level and maybe something about the limitations that brings? Ie that this only really works within a local network.

Reply:
Done